### PR TITLE
Fix selinux perms in Makefile binary build

### PR DESCRIPTION
**Pull request by aweiteka**
As a new contributor I required this change to my environment.

### CODE REVIEW

**Comment by mtrmac** [comment]
Yes, I seem to remember hitting that… but please just use `-v $$(pwd):`… , we don’t need an extra variable.

**Comment by aweiteka** [comment]
> we don’t need an extra variable.

ACK. Updated.

File: Makefile [hunk 1]
```diff
@@ -43,8 +43,8 @@ all: binary docs
 # Then do the build and the output (skopeo) should appear in current dir
 binary: cmd/skopeo
 	docker build ${DOCKER_BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
-	docker run --rm -v ${PWD}:/src/github.com/projectatomic/skopeo \
-		skopeobuildimage make binary-local DEBUG=$(DEBUG)
```
Comment (mtrmac) [inline Makefile]: Why was the `DEBUG=` dropped?

**Comment by runcom** [comment]
LGTM

@aweiteka can you rebase?

File: Makefile [hunk 2]
```diff
@@ -43,7 +43,7 @@ all: binary docs
 # Then do the build and the output (skopeo) should appear in current dir
 binary: cmd/skopeo
 	docker build ${DOCKER_BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
-	docker run --rm -v ${PWD}:/src/github.com/projectatomic/skopeo \
+	docker run --rm -v $$(pwd):/src/github.com/projectatomic/skopeo:Z \
```
Comment (mtrmac) [inline Makefile]: FWIW this `:Z` subsumes #150.  (And I have no objections to that, to be explicit.)

Comment (runcom) [inline Makefile]: I totally missed that Z, I don't have any objection either, if it's ok for @rhatdan we can close #150 and merge this

Comment (rhatdan) [inline Makefile]: I think it would be better to disable selinux for this container then relabeling the skopeo directory.

--security-opt label=disable

**Comment by cgwalters** [comment]
Personally I now use pet containers for development work which have all of the build dependencies installed, and notably here have SELinux disabled because I'm using overlayfs on my workstation.

**Comment by rhatdan** [comment]
You can now run SELinux and overlayfs together if you use a kernel from this repo.

```
cat /etc/yum.repos.d/_copr_pcmoore-kernel-secnext.repo 
[pcmoore-kernel-secnext]
name=Copr repo for kernel-secnext owned by pcmoore
baseurl=https://copr-be.cloud.fedoraproject.org/results/pcmoore/kernel-secnext/fedora-rawhide-$basearch/
skip_if_unavailable=True
gpgcheck=1
gpgkey=https://copr-be.cloud.fedoraproject.org/results/pcmoore/kernel-secnext/pubkey.gpg
enabled=1
```

**Comment by aweiteka** [comment]
@runcom this should be all set.

**Comment by runcom** [comment]
@aweiteka waiting on tests to merge, thx

**Pull request merged**